src/app/GitExtUtils/GitUI/ThreadHelper.cs

```diff
@@ -20,8 +20,48 @@ public static JoinableTaskContext JoinableTaskContext
             internal set => _taskManager = value is null ? null : new(value);
         }
 
+        public static bool IsOnMainThread => JoinableTaskContext?.IsOnMainThread ?? false;
+
         public static JoinableTaskFactory JoinableTaskFactory => _taskManager.JoinableTaskFactory;
 
+        public static void RunJoinableTask(Func<Task> asyncMethod)
```

Comment:
If we keep `RunJoinableTask`, please move to the bottom of the file and omit this unused overload.